Bottom: 67d0afcb74e04c1f9b72bd0c3372289ad9589b5c
Top:    eed4486d2c0ae48fe9844c222ab89a5a5756f077
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-07-17 16:18:26 +0800

Refresh of vhost_net-introduce-tx

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 4ab67f8..2fc4f4b 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -577,7 +577,6 @@ static void handle_tx(struct vhost_net *net)
 			if (unlikely(busyloop_intr)) {
 				vhost_poll_queue(&vq->poll);
 			} else if (unlikely(vhost_enable_notify(&net->dev, vq))) {
-
 				vhost_disable_notify(&net->dev, vq);
 				continue;
 			}
